### DISCUSSION TITLE
How to avoid "Syntax error in text" when using ZenUML

### DISCUSSION THREAD

**Discussion by Eikes-Git** [mermaid-js/mermaid#5971] [Q&A]
**Situation:**
I am currently working on a kind of mermaid editor. The user can enter some mermaid code and that is being rendered via mermaid-js. I am using Gradio for the whole implementation. It works for almost every type of graph I have tried, except for the ZenUML type. If I try to render any ZenUML Graph, I get the following error:

![Screenshot 2024-10-17 105358](https://github.com/user-attachments/assets/8586dd60-0f0c-4d3c-b823-686e67defa80)

**My Code:**
I use the following code to display my mermaid graph:

```html
<!DOCTYPE html>
<html>
<head>
    <title>Mermaid Graph</title>
    <script type="module">
        import mermaid from 'https://cdn.jsdelivr.net/npm/mermaid@11.3/+esm';
        mermaid.initialize({ startOnLoad: true });
    </script>
</head>
<body>
    <div class="mermaid">
        zenuml
            title Declare participant (optional)
            Bob
            Alice
            Alice->Bob: Hi Bob
            Bob->Alice: Hi Alice
    </div>
</body>
</html>
```

Trying to google the problem has not yielded any helpful results that would be useful to me.
I was wondering if using a local instance of the Live Editor could be used to render the graph and I could then display that. However, trying to save a ZenUML Graph with the Live Editor as .PNG or .SVG also does not seem to work correctly. The example code from https://mermaid.js.org/syntax/zenuml.html:

```
zenuml
    title Declare participant (optional)
    Bob
    Alice
    Alice->Bob: Hi Bob
    Bob->Alice: Hi Alice
```

This shows up correctly in the editor, but saving it as .SVG results in a broken graph:

![Screenshot 2024-10-17 110556](https://github.com/user-attachments/assets/d6e792d2-7c0c-427f-bb4c-839e6a39aa6e)

I would be grateful for any information that might help me to display a ZenUML graph.
Thanks in advance for any tips :)

**Comment by CreateAnything** [comment]
<img width="2404" height="1135" alt="image" src="https://github.com/user-attachments/assets/a86a8eab-9baf-43e0-94a4-49a097f678ef" />
Whether this error prompt can be removed globally has affected my page layout
